src/dropdown.ts

```diff
@@ -319,26 +319,17 @@ export default class GlideCoreDropdown
       throw new Error('Only one value is allowed when not `multiple`.');
     }
 
-    // When `value` is set on initial render, this setter is called before
-    // `connectedCallback()` and thus before the default slot has any assigned
-    // elements. So we wait for the update to complete so `this.#optionElements`
-    // isn't empty.
-    this.updateComplete.then(() => {
```

Comment:
> Dropdown and Radio Group no longer reset their `value` after initial render when `value` is set initially

The problem with this approach, instead of the `firstUpdated()` one below, is that this promise resolves after `#onDefaultSlotChange()` is called. And, because there's a `value` but none of the options are selected, `#onDefaultSlotChange()` [resets](https://github.com/CrowdStrike/glide-core/pull/764/files#diff-7431ab16692f7d7a7cfecdb04f55d6c62f87417591065650d88dc407a9b62f3bR1365) `value` to an empty array.

`firstUpdated()`, on the other hand, is called before `#onDefaultSlotChange()`. So `firstUpdated()` is able to select the options present in `value` before `#onDefaultSlotChange()` does its thing with `value`.